com.unity.netcode.gameobjects/Runtime/NetworkVariable/ClientNetworkVariable.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+
+namespace Unity.Netcode
+{
+    /// <summary>
+    /// A ClientNetworkVariable is special in that:
+    ///  - only the owner of the variable can write to it
+    ///  - not even the server can write to it
+    ///  - it is not snapshotted
+    ///  - it must be sent reliably
+    ///
+    /// (This class may be removed in the future when integrated into NetworkVariable natively)
+    /// </summary>
+    [Serializable]
+    public class ClientNetworkVariable<T> : NetworkVariable<T> where T : unmanaged
```

Comment:
would a better name for this class be "OwnerNetworkVariable"? since we're not checking for IsClient but for ownership?